content/docs/conditional-rendering.md

```diff
@@ -8,11 +8,11 @@ redirect_from:
   - "tips/false-in-jsx.html"
 ---
 
-In React, you can create distinct components that encapsulate behavior you need. Then, you can render only some of them, depending on the state of your application.
+React में, आप अलग-अलग कॉम्पोनेन्ट बना सकते हैं जो आपके लिए आवश्यक व्यवहार को संक्षिप्त करते हैं। फिर, आप अपने आवेदन की स्टेट  के आधार पर उनमें से कुछ को ही रिप्रेजेंट  कर सकते हैं।.
```

Comment:
`रिप्रेजेंट  कर सकते हैं।.` => `रिप्रेजेंट कर सकते हैं।`

remove extra space and dot at the end

Reply:
this is also not fixed